generation/spade_gan/spade_gan.ipynb

```diff
@@ -0,0 +1,2665 @@
+{
```

Comment:
<p>Please help modify this markdown, seems not being loaded correctly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Hi @KumoLiu I don't think this comment and the others that you have posted are attached to the cell you're referring to. This seems to be an issue with ReviewNB. Could you please check these and mention which cell this was for by its title or some description? Thanks!

Reply:
<p>Hi, there is still a "[Raw cell]" ahead of the text, please help remove it.</p>